azure_functions_worker/logging.py

```diff
@@ -23,8 +23,15 @@
 
 def format_exception(exception):
     msg = str(exception) + "\n"
-    msg += ''.join(traceback.format_exception(
-        etype=type(exception), value=exception, tb=exception.__traceback__))
+    if sys.version_info.minor < 10:
+        msg += ''.join(traceback.format_exception(
+            etype=type(exception),
+            tb=exception.__traceback__,
+            value=exception))
+    elif sys.version_info.minor == 10:
+        msg += ''.join(traceback.format_exception(exception))
+    else:
+        return exception
```

Comment:
We should remove this return here and let it return msg outside the if condition